helm-org.el

```diff
@@ -62,8 +62,7 @@ Note this have no effect in `helm-org-in-buffer-headings'."
 (defcustom helm-org-headings-actions
   '(("Go to heading" . helm-org-goto-marker)
     ("Open in indirect buffer `C-c i'" . helm-org--open-heading-in-indirect-buffer)
-    ("Refile this heading within this buffer `C-c C-w'" . helm-org-refile-within-buffer-action)
-    ("Refile current or marked heading to selection `C-c w`" . helm-org-heading-refile)
+    ("Refile heading(s) (multiple-marked-to-selected, or current-to-selected) `C-c w`" . helm-org--refile-heading-to)
```

Comment:
Maybe, but I think having the extra explanation may help the discoverability of marking and refiling multiple headings at once.  All I know for sure is that, after several years, it wasn't until you accepted my previous PR and you reviewed it that I understood what `Refile current or marked heading to selection` meant.  ;)  (That's not a criticism; it's very hard to get the idea across clearly in a few words.)